features/steps/steps.go

```diff
@@ -107,7 +107,7 @@ func (c *Component) filterOutputIsInDatastore(expectedOutput *godog.DocString) e
 
 	err := json.Unmarshal([]byte(expectedOutput.Content), &expected)
 	if err != nil {
-		return errors.Wrap(err, "failed to unmarshall provided filterOutput")
+		return fmt.Errorf("failed to unmarshall provided filterOutput: %w", err)
```

Comment:
I had a question about the use of fmt.Errorf() in this file: I'm assuming you didn't want githum.com/pkg/errors/Wrap() to avoid the storing of the stack frame. But would it not be better to use githum.com/pkg/errors/WrapMessage() instead of fmt.Errorf() - it's confusing switching between the use of different packages for error creation, especially since githum.com/pkg/errors has an Errorf() function which does not wrap an error, and so compounds teh confusion in the switch to fmt.Errorf()

Reply:
Ah I just remembered why I changed it actually, it's not just to remove the stack trace it was that the output wasn't being shown properly. I think it's something to do with the way the GoDog packages work but only the wrapping message was being shown, not the underlying error.